graphql-schemas/schema.yml

```diff
@@ -1169,6 +1169,25 @@ confs:
   - { name: upgradePolicy, type: ClusterUpgradePolicy_v1 }
   - { name: upgradePolicyTemplate, type: OpenShiftClusterManagerUpgradePolicyTemplate_v1 }
 
+- name: FleetSubscriptionLabelTemplate_v1
+  fields:
+  - { name: path, type: string, isRequired: true, isResource: true, resolveResource: true }
+  - { name: type, type: string }
+  - { name: variables, type: json }
+
+- name: FleetLabelDefault_v1
+  fields:
+  - { name: name, type: string, isRequired: true, isUnique: true }
+  - { name: matchSubscriptionLabels, type: json, isRequired: true }
+  - { name: subscriptionLabelTemplate, type: FleetSubscriptionLabelTemplate_v1 }
```

Comment:
This attribute is required in the schema but not here. Please also add `isRequired: true`.

Reply:
Oh great catch! Fixed :+1: